database/src/main/java/Main.java

```diff
@@ -1,8 +1,6 @@
-package database;
```

Comment:
Please create package `database` in `java` folder and move all the code inside this package. Don't forget to update entity path in `controller`.